Readme.md

```diff
@@ -0,0 +1,155 @@
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LIVE DEMO -->
+ 
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+- [<a href="https://tracy-miranja.github.io/HTML-CSS-JavaScript-capstone-project/">Clich here to see Live Demo]
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- GETTING STARTED -->
+ 
+## 💻 Getting Started <a name="getting-started"></a>
+ 
+To get a local copy up and running, follow these steps.
+ 
+### Prerequisites
+ 
+In order to run this project you need:
+ 
+- A computer.
+- Internet access.
+- A modern web browser.
+ 
+### Setup
+ 
+Clone this repository to your desired folder:
+ 
+```sh
+  cd my-folder
+  git clone git@github.com:Tracy-miranja/HTML-CSS-JavaScript-capstone-project.git
+```
+ 
+### Install
+ 
+Install this project with:
+ 
+```sh
+  cd my-folder
+  npm-install
+```
+### Usage
+ 
+To run the project, open index.html in your text editor,right-click and open with live server,it will be opened in your web browser.
+ 
+<!-- AUTHORS -->
+ 
+## 👥 Author <a name="author"></a>
+ 
+ 
+👤 **Tracy miranja**
+ 
+- GitHub: [@githubhandle](https://github.com/Tracy-miranja)
+- Twitter: [@twitterhandle](https://twitter.com/tracymiranja)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/tracymiranja)
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- FUTURE FEATURES -->
+ 
+## 🔭 Future Features <a name="future-features"></a>
+ 
+- [ ] **Web page header**
+- [ ] **Web page body**
+- [ ] **Web page footer**
+- [ ] **Desktop design**
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Key FEATURES -->
+
+- [ ] **Homepage**
+- [ ] **About Page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- SUPPORT -->
+ 
+## ⭐️ Show your support <a name="support"></a>
+ 
+If the project was great as your view, don't hesitate to share and give it a star.
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- ACKNOWLEDGEMENTS -->
+ 
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+ 
+I would like to thank to all my Mricronauts partners for support and microverse for giving me this design template to work on.
+i would also like to thank Cindy Shin in Behance for the ui/ux design in behance.
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LICENSE -->
+ 
+## 📝 License <a name="license"></a>
+ 
+This project is [MIT](https://github.com/Tracy-miranja/HTML-CSS-JavaScript-capstone-project/compare/main...add-license-1 licensed.
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+README.md
```

Comment:
- [ ] I think it would be better to provide a direct link to the MIT license life, not the PR link to make it more professional 😀😎✨
   _Kindly make this small adjustment and make sure that your License file is available in your Root directory 🙏_

Reply:
- [ ] Fixed. 
- [Optional ] Instead of use MIT license link inside brace you can use just like this [MIT](https://github.com/Tracy-miranja/HTML-CSS-JavaScript-capstone-project/blob/add-license-1/LICENCE.md ) with this `[MIT](https://github.com/Tracy-miranja/HTML-CSS-JavaScript-capstone-project/blob/add-license-1/LICENCE.md ) `

Reply:
Thanks for the code review and the feedback @Abenezer-Tilahun and @TuyishimireEric TuyishimireEric. I'll work on the feedback and try to reach out if stuck.